app/javascript/src/Plans/components/PlansTable.tsx

```diff
@@ -79,27 +79,27 @@ const PlansTable: FunctionComponent<Props> = ({
     })
     .catch(err => {
       console.error(err)
-      flash.error('An error ocurred. Please try again later.')
+      toast('An error occurred. Please try again later.', 'danger')
     })
     .finally(() => { setIsLoading(false) })
 
   const handleActionPublishHide = (path: string) => ajax<{ notice: string; plan: string; error: string }>(path, { method: 'POST' })
     .then(data => data.json()
       .then((res) => {
         if (data.status === 200) {
-          flash.notice(res.notice)
+          toast(res.notice, 'success')
```

Comment:
So, I find this tranformation confusing :/

The controller still sets `:notice`, and it was not straightforward to figure out why a "success" toast was shown.

I think we need to replace ALL `flash[:notice]` to `flash[:success]`, ans so simplify this part.

It's OK if it's done in another PR.